Hcal/include/Hcal/HcalDigiProducer.h

```diff
@@ -84,6 +84,10 @@ class HcalDigiProducer : public framework::Producer {
   /// development
   bool noise_{true};
 
+  /// If false, save digis from all channels, even pure noise in empty bars
+  /// Helpful when comparing with test beam data
+  bool zeroSuppression_;
```

Comment:
I think we should have a default value here
```suggestion
  bool zeroSuppression_{true};
```